src/test/java/org/example/TaskServiceTest.java

```diff
@@ -0,0 +1,68 @@
+package org.example;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class TaskServiceTest {
+    TaskService taskService;
+
+    @BeforeEach
+    void setUp() {
```

Comment:
Wydaje mi sie, że dużo lepiej by się te testy czytało jakby serwis był tworzony za każdym razem z jakimiś taskami, bo teraz to dla każdego testu muszę scrolować do góry. Np. `new TaskService(List.of(new Task("1", false)))` jedna linia